mos.yml

```diff
@@ -31,7 +31,7 @@ config_schema:
   - ["wifi.sta_all_chan_scan", true]
 
   - ["shelly.name", "s", "", {title: "User-facing device name"}]  # Note: will be overridden by cfg 2 -> 3 migration.
-  - ["shelly.mode", "i", 0, {title: "System mode: 0 - default, 1 - roller shutter (if supported), 2 - garage door opener (if supported)"}]
+  - ["shelly.mode", "i", 0, {title: "System mode: 0 - default, 1 - roller shutter (if supported), 2 - garage door opener (if supported), 3 - RGB (if supported), 4 - RGBW (if supported)"}]
```

Comment:
can Shelly RGBW2 be a roller shutter or a garage door opener? probably not.
can Shelly 2.5 be an RGBW PWM controller? definitely ot.

RGB is not a mode, it should be the only component instantiated for RGBW2 device in shelly_init, and no other device can be using it.
RGBW is a mode for the RGB component, it should be a boolean option.

Reply:
We will add later a 4xW mode and RGB will be renamed  to RGB+W so that you can use RGB as now and the W as a different channel, so the mode creates 1, 2 or 3 components. I unterstand your meaning at the moment, but on the long term mode is the right place.

Reply:
Fun fact: RGBW2 as garage door opener works, just change the pins and flash the 2.5 firmware onto the RGBW2

Reply:
ok, makes sense.